step.yml

```diff
@@ -0,0 +1,84 @@
+title: Appetize.io deploy
+summary: Deploy the app to Appetize.io
+description: |-
+  Deploy your app to Appetize.io and run it in your browser.
+  For app demos, training, customer support, development, testing, and more.
+website: https://github.com/bitrise-steplib/steps-appetize-io-deploy
+source_code_url: https://github.com/bitrise-steplib/steps-appetize-io-deploy
+support_url: https://github.com/bitrise-steplib/steps-appetize-io-deploy/issues
+
+project_type_tags:
+  - ios
```

Comment:
if all of our platforms can use this, please leave this list empty rather than list all of them